### PR TITLE
2.2 announcment

### PR DESCRIPTION
**Pull request by corinnekrych**
@matzew mind to merge when appropriate?

### CODE REVIEW

File: _posts/2015-03-16-aerogear-ios-2.2.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The work around synchronisation on iOS is covered with:
+
+- [aerogear-ios-sync](https://github.com/aerogear/aerogear-ios-sync): the sync engine is where the algorithm is implemented. The sync algorithm is based on Google's [Differential Synchronisation](http://research.google.com/pubs/pub35605.html). We provide two implementations: one text-based [DiffMatchPatch](https://github.com/aerogear/aerogear-diffmatchpatch-ios) and a JsonPatch version based on [JSON Patch RFC6902](https://tools.ietf.org/html/rfc6902). 
+
+- [aerogear-ios-sync-client](https://github.com/aerogear/aerogear-ios-sync-client): the client uses sync engine and a network implementation. Our libraries is based on WebSockets to provide real-time sync.
+
+- [Jedi demo](https://github.com/aerogear/aerogear-ios-cookbook/tree/master/Jedi): Star war fan, rejoice in discovering our two Jedi hobbies demos (DiffMatchPatch and JsonPatch).
```
Comment (matzew) [inline _posts/2015-03-16-aerogear-ios-2.2.md]: Star Wars, instead of war ?

File: _posts/2015-03-16-aerogear-ios-2.2.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      
+**2. House cleaning tasks:**
+
+- Move to [cocoapods 0.36.0](http://blog.cocoapods.org/CocoaPods-0.36/) for using iOS dynamic framework.
+
+- Deprecate [aerogear-ios-httpstub](https://github.com/aerogear/aerogear-ios-httpstub).
```
Comment (matzew) [inline _posts/2015-03-16-aerogear-ios-2.2.md]: let's add some more context here, deprecated in fav. of OHHTTPSsub + a link or something?

**Comment by matzew** [comment]
@corinnekrych landed

File: _posts/2015-03-16-aerogear-ios-2.2.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: AeroGear iOS SDK v2.2 is out !
```
Comment (lfryc) [inline _posts/2015-03-16-aerogear-ios-2.2.md]: The post is missing author and categorization, see https://raw.githubusercontent.com/aerogear/aerogear.org/master/_posts/2015-01-30-aerogear-ios-2.1.md for inspiration.

Comment (lfryc) [inline _posts/2015-03-16-aerogear-ios-2.2.md]: Okay, Andres already sent and update in https://github.com/aerogear/aerogear.org/pull/565

**Pull request merged**